dds/DCPS/transport/rtps_udp/RtpsUdpSendStrategy.cpp

```diff
@@ -355,6 +425,13 @@ namespace {
                  ex.message.in()));
     }
   }
+
+  void check_stateless_volatile(EntityId_t writerId, bool& stateless_or_volatile)
```

Comment:
I see this being called with both sender and receiver EntityIds below. Should we be checking against the STATELESS_READER and _VOLATILE_SECURE_READER EntityIds here as well?

Reply:
The "sender" call is for the writer messages like Data and Heartbeat, while the "receiver" call is for AckNack (etc.) so in either case they are writers.